pkg/cmd/volume/ls.go

```diff
@@ -48,18 +48,48 @@ func Ls(options *types.VolumeLsCommandOptions, stdout io.Writer) error {
 		logrus.Warn("cannot use --size and --quiet together, ignoring --size")
 		options.Size = false
 	}
-	labelFilterFuncs, nameFilterFuncs, sizeFilterFuncs, isFilter, err := getVolumeFilterFuncs(options.Filters)
-	if err != nil {
-		return err
-	}
-	if len(sizeFilterFuncs) > 0 && options.Quiet {
+	sizeFilter := hasSizeFilter(options.Filters)
+	if sizeFilter && options.Quiet {
 		logrus.Warn("cannot use --filter=size and --quiet together, ignoring --filter=size")
-		sizeFilterFuncs = nil
+		options.Filters = removeSizeFilters(options.Filters)
 	}
-	if len(sizeFilterFuncs) > 0 && !options.Size {
+	if sizeFilter && !options.Size {
 		logrus.Warn("should use --filter=size and --size together")
 		options.Size = true
 	}
+	vols, err := List(
```

Comment:
maybe an empty before the `List` call, to separate the option process and other logic

Reply:
[25314c8](https://github.com/davidhsingyuchen/nerdctl/pull/2/commits/25314c820d7e87507ac2bedab70a98241b79abcc)